### DISCUSSION TITLE
Where is the temporary folder for requesting cache dump?

### DISCUSSION THREAD

**Discussion by YSYHZ** [Sagnac/streamsave#16] [Q&A] [answered]
Sorry, I don't know English. The following content was generated using translation
When watching bdmv, I accidentally pressed the shortcut key for the output file. bdmv is too large by 40 GB, and streamsave reported an error that the file was not written out. But the temporary files in the middle were not cleared
The C drive is directly full, and there was still 10 Gb of remaining space left before.
Now I would like to ask where the temporary folder is or what effective cleaning methods are available?
If I have disturbed you, I am very sorry

**Comment by Sagnac** [comment, marked as answer]
Hi, there is no temporary cache directory, just the one defined with `save_directory`, which is where all the files should be written to; this defaults to your working directory if not specified. The console should tell you where the file was written to if it was.

**Reply by YSYHZ** [reply]
Thank you very much for your answer
